quipucords/scanner/manager.py

```diff
@@ -39,6 +39,20 @@ def __init__(self):
         self.running = True
         logger.debug('Scan manager created.')
 
+    def log_info(self):
+        """Log the status of the scan manager."""
+        interval = 60
+        heartbeat = Timer(interval, self.log_info)
+        logger.debug('Scan manager running: %s. '
+                     'Current task: %s. '
+                     'Number of scans in queue: %s.',
+                     self.running, self.current_task, len(self.scan_queue))
+        if self.running:
+            heartbeat.start()
+        else:
+            if heartbeat.is_alive():
+                heartbeat.cancel()
```

Comment:
I don't understand this else condition. How could heartbeat be alive? Its a local variable.

Reply:
Why do we need a new timer? Can't the manager just log in the run loop?